<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>350</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Auto Clicker</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSlider" name="sleep_up_slider">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>66</y>
      <width>301</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Fluent Icons</family>
     </font>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDigitsOnly</set>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>1998</number>
    </property>
    <property name="singleStep">
     <number>5</number>
    </property>
    <property name="value">
     <number>200</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="invertedControls">
     <bool>false</bool>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
    <property name="tickInterval">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>18</y>
      <width>161</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Fluent Icons</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Задержка от (мс):</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>134</y>
      <width>161</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Fluent Icons</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Задержка до (мс):</string>
    </property>
   </widget>
   <widget class="QSlider" name="sleep_to_slider">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>184</y>
      <width>301</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Fluent Icons</family>
     </font>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDigitsOnly</set>
    </property>
    <property name="minimum">
     <number>3</number>
    </property>
    <property name="maximum">
     <number>2000</number>
    </property>
    <property name="singleStep">
     <number>5</number>
    </property>
    <property name="value">
     <number>300</number>
    </property>
    <property name="tracking">
     <bool>true</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
    <property name="tickInterval">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_cpm">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>231</y>
      <width>301</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Fluent Icons</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>4.0 CPS</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="start_btn">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>402</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Fluent Icons</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Включить</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="sleep_up_lineEdit">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>18</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Fluent Icons</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDigitsOnly</set>
    </property>
    <property name="text">
     <string>200</string>
    </property>
    <property name="maxLength">
     <number>4</number>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="sleep_to_lineEdit">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>134</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Fluent Icons</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDigitsOnly</set>
    </property>
    <property name="text">
     <string>300</string>
    </property>
    <property name="maxLength">
     <number>4</number>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_clicks">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>468</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Fluent Icons</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0 Кликов</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_cycle_clicks">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>280</y>
      <width>151</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Fluent Icons</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Кликов за цикл:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_cycle_clicks">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>280</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Fluent Icons</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="value">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_cycle_clicks_2">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>340</y>
      <width>171</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Fluent Icons</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Клавиша для бинда:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_bind_pushButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>340</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
